File: chapter_4/vb/code_1/code1_public.vb
Module code1_public

    Class Box
        Public length As Double
        Public breadth As Double
        Public height As Double

        Public Sub setLength(ByVal len As Double)
            length = len
        End Sub

        Public Sub setBreadth(ByVal bre As Double)
            breadth = bre
        End Sub

        Public Sub setHeight(ByVal hei As Double)
            height = hei
        End Sub

        Public Function getVolume() As Double
            Return length * breadth * height
        End Function
    End Class

    Sub Main()
        Dim Box1 As New Box
        Dim Box2 As New Box
        Box1.setLength(9)
        Box1.setBreadth(5)
        Box1.setHeight(10)

        Box2.setLength(14)
        Box2.setBreadth(15)
        Box2.setHeight(12)

        Console.WriteLine("Volume of Box1 is " & Box1.getVolume)
        Console.WriteLine("Volume of Box2 is " & Box2.getVolume & vbCrLf)

    End Sub
End Module

File: chapter_4/Csharp/Csharp/code1_public.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Csharp
{
    class Box
    {
        public double length;
        public double breadth;
        public double height;

        public void setLength(double len)
        {
            length = len;
        }

        public void setBreadth(double bre)
        {
            breadth = bre;
        }

        public void setHeight(double hei)
        {
            height = hei;
        }

        public double getVolume()
        {
            return length * breadth * height;
        }
    }

    class code1_public
    {
        
        public static void display()
        {
            Box box1 = new Box();
            Box box2 = new Box();
            box1.setLength(10);
            box1.setHeight(12);
            box1.setBreadth(5);
            box2.setLength(15);
            box2.setHeight(17);
            box2.setBreadth(9);
            Console.WriteLine("Volume of Box1 is " + box1.getVolume());
            Console.WriteLine("Volume of Box2 is " + box2.getVolume() + "\n");
        }
    }
}
